daemon/src/model.rs

```diff
@@ -13,7 +13,7 @@ use uuid::Uuid;
 
 pub mod cfd;
 
-#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq, sqlx::Type)]
```

Comment:
I think this is not actually doing anything because we are not specifying `#[sqlx(transparent)]`.

Can you remove it to avoid confusion about what this is doing?

Reply:
Oops! Yes, removed now.